docs/BR.md

```diff
@@ -1693,8 +1693,13 @@ The CA SHALL reject a certificate request if one or more of the following condit
 1. The Key Pair does not meet the requirements set forth in [Section 6.1.5](#615-key-sizes) and/or [Section 6.1.6](#616-public-key-parameters-generation-and-quality-checking);
 2. There is clear evidence that the specific method used to generate the Private Key was flawed;
 3. The CA is aware of a demonstrated or proven method that exposes the Applicant's Private Key to compromise;
-4. The CA has previously been made aware that the Applicant's Private Key has suffered a Key Compromise, such as through the provisions of [Section 4.9.1.1](#4911-reasons-for-revoking-a-subscriber-certificate);
-5. The CA is aware of a demonstrated or proven method to easily compute the Applicant's Private Key based on the Public Key (such as a Debian weak key, see <https://wiki.debian.org/SSLkeys>).
+4. The CA has previously been notified that the Applicant's Private Key has suffered a Key Compromise using the CA's procedure for revocation request as described in [Section 4.9.3](#493-procedure-for-revocation-request);
```

Comment:
I think this should also reference  Section 4.9.12 "Special requirements re key compromise" of the CA's CP and/or CPS.

There's a big difference between someone emailing cert-prob-reports@ (or similar documented revocation procedure) saying "please revoke this certificate because its key has been compromised" and someone emailing cert-prob-reports@ with a CSR signed by that private key and a CN of "this key has been compromised". Equivalently for ACME, there's a big difference between a revoke-cert request signed by the subscriber key specifying the keyCompromise revocation reason, and a revoke-cert request signed by the certificate key itself. In both cases, the former is just a revocation request, while the latter is a _demonstration_ of key compromise.

I interpret "made aware that the Applicant's Private Key has suffered a Key Compromise" as requiring a demonstration of said compromise -- simply asserting it to me isn't enough. But it's much harder to interpret "notified ... using the CA's procedure for revocation request" in the same way. That sounds like I'd have to treat both simple revocation requests and true demonstrations of compromise in the same way, and block all future issuance for both.

Therefore I think a reference to 4.9.12 of the CP/CPS is necessary.

Reply:
Makes sense and I added that reference. Thanks!